src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/SampleData.java

```diff
@@ -38,23 +43,39 @@
         .caseTypeId(CASE_TYPE_ID)
         .build();
 
-    public static String envelopeJson() throws Exception {
-        return new JSONObject()
-            .put("id", "eb9c3598-35fc-424e-b05a-902ee9f11d56")
-            .put("case_ref", CASE_REF)
-            .put("jurisdiction", JURSIDICTION)
-            .put("zip_file_name", "zip-file-test.zip")
-            .put("classification", Classification.NEW_APPLICATION)
-            .put("documents", new JSONArray()
-                .put(new JSONObject()
-                    .put("file_name", "hello.pdf")
-                    .put("control_number", "control_number")
-                    .put("type", "doc_type")
-                    .put("scanned_at", toIso8601(Instant.EPOCH))
-                    .put("url", "https://example.gov.uk/123")
+    public static String exampleJson = fromFile("envelopes/example.json");
+
+    public static String envelopeJson() {
+        try {
+            return new JSONObject()
+                .put("id", "eb9c3598-35fc-424e-b05a-902ee9f11d56")
+                .put("case_ref", CASE_REF)
+                .put("jurisdiction", JURSIDICTION)
+                .put("zip_file_name", "zip-file-test.zip")
+                .put("classification", Classification.NEW_APPLICATION)
+                .put("documents", new JSONArray()
+                    .put(new JSONObject()
+                        .put("file_name", "hello.pdf")
+                        .put("control_number", "control_number")
+                        .put("type", "doc_type")
+                        .put("scanned_at", toIso8601(Instant.EPOCH))
+                        .put("url", "https://example.gov.uk/123")
+                    )
                 )
-            )
-            .toString();
+                .toString();
+        } catch (Exception e) {
+            throw new RuntimeException("Could not make envelopeJson", e);
```

Comment:
added this so that we don't have to keep puting Exception on the test methods.

Reply:
we always do anyway

Reply:
Its just more boilerplate and cumbersome. Why if we dont need it ?